.travis.yml

```diff
@@ -0,0 +1,9 @@
+language: python
+python:
+    - "2.7"
```

Comment:
Clarification: you don't need to make all of the code Python 3.x-compatible with this PR, but we should still be testing to ensure compatibility.